ci/checks/style.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Copyright (c) 2018, NVIDIA CORPORATION.
+#####################
+# cuDF Style Tester #
+#####################
+
+# Ignore errors and set path
+set +e
+PATH=/conda/bin:$PATH
+
+# Activate common conda env
+source activate gdf
+
+# Run flake8 and get results/return code
+FLAKE=`flake8 python`
```

Comment:
Looks like this file still only runs `flake8`. Where are `black` & `isort`?

Reply:
I think that we should also run `black` in the style tests